### PR TITLE
github actions workflows

### PR DESCRIPTION
**Pull request by ulfox**
Green tests
    Pep8 linting/formatting
    github actions run the tests automatically

### CODE REVIEW

File: .github/workflows/flake_eks_workflow_test.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        cd ./aws/eks/
+        python -m pip install --upgrade pip
```
Comment (spirosoik) [inline .github/workflows/flake_eks_workflow_test.yml:19]: make install (upgrade pip, install flake8)

File: .github/workflows/flake_eks_workflow_test.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        pip install -r lambdas/requirements.txt
+    - name: Lint with flake8
+      run: |
+        cd ./aws/eks/
+        pip install flake8
+        flake8 lambdas/lambdas.py \
```
Comment (spirosoik) [inline .github/workflows/flake_eks_workflow_test.yml:25]: make lint (runs flake8)

File: .github/workflows/flake_eks_workflow_test.yml (hunk 1 above)
Comment (ulfox) [reply .github/workflows/flake_eks_workflow_test.yml]: fixed

File: .github/workflows/flake_eks_workflow_test.yml (hunk 2 above)
Comment (ulfox) [reply .github/workflows/flake_eks_workflow_test.yml]: fixed

**Pull request merged**